src/components/editors/backgroundEditor.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+
+import { Grid, Typography } from '@material-ui/core'
+
+import { OPTIONS } from '../../lib/options'
+
+import { Dropdown, ColorPicker, PopoverIcon } from '../SettingComponent'
+
+const BACKGROUND_EDITOR_OPTIONS = [
+  { name: 'Overlay', color: OPTIONS.backgroundColor, size: OPTIONS.backgroundSize },
+  { name: 'Window', color: OPTIONS.backgroundWindowColor, size: OPTIONS.backgroundWindowSize },
+  { name: 'Text', color: OPTIONS.backgroundTextColor, size: OPTIONS.backgroundTextSize },
+]
+
+const BackgroundEditor = () => (
+  <Grid
+    container
+    direction="column"
+    justify="flex-start"
+    alignItems="flex-start"
+    spacing={5}
+  >
+    {BACKGROUND_EDITOR_OPTIONS.map( opt => (
+      <Grid item>
+        <Typography key={opt.name} align="center">{opt.name}</Typography>
+        <div className="someOption">
+          <Typography className="someOptionName">{opt.color.name}</Typography>
+          <PopoverIcon
+            icon={opt.color.icon}
+            component={(
+              <ColorPicker
+                name={opt.color.name}
+                storageKey={opt.color.storageKey}
+              />
+              )}
+          />
+        </div>
+        <div className="someOption">
```

Comment:
Newline between these div separators of components (think of it like english paragraphs, a break makes it easier when there's a new topic about to be discussed)